### DISCUSSION TITLE
HyperHDR + WLED (can't change led layout)

### DISCUSSION THREAD

**Discussion by kapellmeister84** [awawa-dev/HyperHDR#435] [Q&A]
Hey guys, I'm using WLED and HyperHDR and the LEDs light up nicely. But they are upside down and mirrored. No matter what I try, I can't change the LED Layout. It won't update, nor save. Any ideas? All software is up to date. Thanks!

**Comment by awawa-dev** [comment]
Hi
do you have 'reverse direction' enabled in the led strip configuration?

**Comment by kapellmeister84** [comment]
Yep. I can’t change anything in the led layout panel. I mean.. I can, but it doesn’t reflect on the LEDs...
